app/controllers/events_controller.rb

```diff
@@ -0,0 +1,112 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: %i[show edit update destroy]
+  before_action :set_current_user, only: %i[show index new attend_events show_events]
+
+  # GET /events
+  # GET /events.json
+  def index
+    @events = Event.all
```

Comment:
I believe you don't need variable if you are already showing future and past events. Don't repeat the same list.